.changeset/tiny-cobras-look.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-common': minor
+---
+
+Expose an ``onAuth` handler for `git` actions to provide custom credentials
```

Comment:
```suggestion
'@backstage/backend-common': patch
---

Expose an `onAuth` handler for `git` actions to provide custom credentials
```

Fix from above + switch to patch

Reply:
Ah alright, don't have permission to apply this change. You'll need to apply it yourself

Reply:
Changed to `patch` 👍